pep508.nix

```diff
@@ -160,12 +160,12 @@ let
       hasElem = needle: haystack: builtins.elem needle (builtins.filter (x: builtins.typeOf x == "string") (builtins.split " " haystack));
       op = {
         "true" = x: y: true;
-        "<=" = x: y: (unmarshal x) <= (unmarshal y);
-        "<" = x: y: (unmarshal x) < (unmarshal y);
+        "<=" = x: y: op.">=" y x;
```

Comment:
Would `op."<=" x y;` be clearer?

Reply:
I think that would cause infinite recursion right?

Reply:
Oh, is `op.">="` the same as `>=`? In that case I'm not sure what the final bindings end up as. We might want a bunch more tests to make sure they all work as expected.

Reply:
I think it's not as complex as you might think. What's happening here is analogous to the following code in `nix repl`:

```
nix-repl> let op = { gt = x: y: x > y; lt = x: y: op.gt y x; }; in op.lt 1 2
true
```